srtcore/congctl.cpp

```diff
@@ -498,7 +498,7 @@ class FileCC : public SrtCongestionControlBase
         HLOGC(cclog.Debug, log << "FileCC: LOSS: "
             << "sent=" << CSeqNo::seqlen(m_iLastAck, m_parent->sndSeqNo()) << ", inFlight=" << pktsInFlight
             << ", lost=" << numPktsLost << " ("
-            << lost_pcent_x10 / 10 << "." << lost_pcent_x10 % 10 << "\%)");
+            << lost_pcent_x10 / 10 << "." << lost_pcent_x10 % 10 << "%%)");
```

Comment:
Are you sure that this shouldn't be in parentheses?
```
<< (lost_pcent_x10 / 10) << "." << (lost_pcent_x10 % 10) << "%)");
```

BTW is this double % needed? This facility doesn't use a printf-format.

Reply:
Hm, indeed...